src/common/Pagination/Pagination.tsx

```diff
@@ -48,7 +54,7 @@ export const Pagination: FC<PaginationProps> = forwardRef<any, PaginationProps>(
   };
 
   return (
-    <Block name="pagination" mod={{ size }}>
+    <Block name="pagination" mod={{ size, outline, align, noPadding }}>
```

Comment:
looks like too much parameters for tag used just twice. if the problem is in different designs maybe it's better to get them in sync?

Reply:
i prefer to be explicit. i could of course use single param like "bare=true" or somethig that combines these three. yet, it won't describe itself and I'll have to add documentation. also, these parameters might be useful in other pagination cases